c-ares-sys/src/ffi.rs

```diff
@@ -455,6 +458,7 @@ pub struct ares_addr6ttl {
 }
 #[repr(C)]
 #[derive(Debug, Copy, Clone)]
+#[cfg(cares1_17)]
```

Comment:
this one should be unnecessary, right?  

- if c-ares is post-1.17 then of course there is no problem
- if c-ares is pre-1.17 then users can construct this structure if they really want to; but there is no way for them to interact with c-ares using it